deku-p/src/core/crypto/BLAKE2b.ml

```diff
@@ -18,6 +18,16 @@ struct
   let of_raw string = of_raw_string_opt string
   let to_raw hash = to_raw_string hash
 
+  let encoding =
```

Comment:
This is not correct:
```
2: { "status": "error",
2:   "error":
2:     [ "Unknown",
2:       "BytesValidationError: [block_payload_hash] Invalid bytes: [object Object]\n    at BytesToken.isValid (webpack://sidechain/../../../../../../../../../../nix/store/xq6zg7327z578ns4i9783z9snazgkrq3-sidechain-0.0.0/lib/node_modules/sidechain/node_modules/@taquito/michelson-encoder/dist/taquito-michelson-encoder.es6.js?:1414:20)\n    at BytesToken.Encode (webpack://sidechain/../../../../../../../../../../nix/store/xq6zg7327z578ns4i9783z9snazgkrq3-sidechain-0.0.0/lib/node_modules/sidechain/node_modules/@taquito/michelson-encoder/dist/taquito-michelson-encoder.es6.js?:1423:26)\n    at eval (webpack://sidechain/../../../../../../../../../../nix/store/xq6zg7327z578ns4i9783z9snazgkrq3-sidechain-0.0.0/lib/node_modules/sidechain/node_modules/@taquito/michelson-encoder/dist/taquito-michelson-encoder.es6.js?:646:46)\n    at Array.map (<anonymous>)\n    at PairToken.Encode (webpack://sidechain/../../../../../../../../../../nix/store/xq6zg7327z578ns4i9783z9snazgkrq3-sidechain-0.0.0/lib/node_modules/sidechain/node_modules/@taquito/michelson-encoder/dist/taquito-michelson-encoder.es6.js?:646:33)\n    at eval (webpack://sidechain/../../../../../../../../../../nix/store/xq6zg7327z578ns4i9783z9snazgkrq3-sidechain-0.0.0/lib/node_modules/sidechain/node_modules/@taquito/michelson-encoder/dist/taquito-michelson-encoder.es6.js?:646:46)\n    at Array.map (<anonymous>)\n    at PairToken.Encode (webpack://sidechain/../../../../../../../../../../nix/store/xq6zg7327z578ns4i9783z9snazgkrq3-sidechain-0.0.0/lib/node_modules/sidechain/node_modules/@taquito/michelson-encoder/dist/taquito-michelson-encoder.es6.js?:646:33)\n    at eval (webpack://sidechain/../../../../../../../../../../nix/store/xq6zg7327z578ns4i9783z9snazgkrq3-sidechain-0.0.0/lib/node_modules/sidechain/node_modules/@taquito/michelson-encoder/dist/taquito-michelson-encoder.es6.js?:646:46)\n    at Array.map (<anonymous>) {\n  value: [ { invalid_utf8_string: [Array] } ],\n  token: BytesToken {\n    val: { prim: 'bytes', annots: [Array] },\n    idx: 1,\n    fac: [Function: createToken],\n    createToken: [Function: createToken]\n  }\n}" ] }deku-node: [ERROR] 
2: inject: Json_encoding.Cannot_destruct(_)
```

I think it should use `of_b58` and `to_b58` but not sure on how to do that...